aiida/cmdline/params/options/commands/code.py

```diff
@@ -91,6 +91,15 @@ def is_not_on_computer(ctx):
     help='Name of the computer, on which the code is installed.'
 )
 
+WITHMPI = OverridableOption(
+    '--withmpi/--withoutmpi',
```

Comment:
For clarity I think it is always better to separate words with hyphens
```suggestion
    '--with-mpi/--without-mpi',
```